README.md

```diff
@@ -154,29 +154,29 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 | :-- | :-- | :-: | :-: |
 | [Angular module](https://github.com/avmaisak/ngx-simple-icons) <img src="https://cdn.simpleicons.org/angular/000/fff" alt="Angular" align=left width=24 height=24> | [@avmaisak](https://github.com/avmaisak) | ![](https://img.shields.io/github/license/avmaisak/ngx-simple-icons) | [![v6.18.0](https://img.shields.io/static/v1?label=version&message=v6.18.0&logo=simpleicons&color=red)](https://github.com/simple-icons/simple-icons/tree/6.18.0) |
 | [Blazor Nuget package](https://github.com/TimeWarpEngineering/timewarp-simple-icons) <img src="https://cdn.simpleicons.org/blazor/000/fff" alt="Blazor" align=left width=24 height=24> | [@TimeWarpEngineering](https://github.com/TimeWarpEngineering)  | ![](https://img.shields.io/github/license/TimeWarpEngineering/timewarp-simple-icons) | [![v9.9.0](https://img.shields.io/static/v1?label=version&message=v9.9.0&logo=simpleicons&color=yellow)](https://github.com/simple-icons/simple-icons/tree/9.9.0) |
```

Comment:
As you can see in the ["Steps to publish NuGet package" section](https://github.com/TimeWarpEngineering/timewarp-simple-icons#steps-to-publish-nuget-package), the version of the `.csproj` file is manually maintained in sync with the simple-icons *package.json* `version`. As *.csproj* files are XML, we can dynamize this badge.

```suggestion
| [Blazor Nuget package](https://github.com/TimeWarpEngineering/timewarp-simple-icons) <img src="https://cdn.simpleicons.org/blazor/000/fff" alt="Blazor" align=left width=24 height=24> | [@TimeWarpEngineering](https://github.com/TimeWarpEngineering)  | ![](https://img.shields.io/github/license/TimeWarpEngineering/timewarp-simple-icons) | [![](https://img.shields.io/badge/dynamic/xml?url=https%3A%2F%2Fraw.githubusercontent.com%2FTimeWarpEngineering%2Ftimewarp-simple-icons%2Fmain%2Fsource%2Ftimewarp-simple-icons%2Ftimewarp-simple-icons.csproj&query=%2FProject%2FPropertyGroup%2FVersion&logo=simpleicons&label=version)](https://github.com/simple-icons/simple-icons/tree/9.9.0) |
```